packages/my-react-package/src/ContractWithWallet.tsx

```diff
@@ -89,37 +95,50 @@ class ContractWithWallet {
                 type: functionABI.inputs[index].type
             }
         });
-        const {safeTxBody,scwAddress}= await this.buildExecTx(this.contract.chain[this.chain].address,name,args);
-        const signedTx=await this.wallet.getSignedTx(scwAddress,this.chain,safeTxBody);
-        this.sendSignedTransaction(signedTx);
-        
+
+        const { safeTxBody, scwAddress } = await this.buildExecTx(this.contract.chain[this.chain].address, name, args);
+
+        const signedTx = await this.wallet.getSignedTx(scwAddress, this.chain, safeTxBody);
+
+        const nonce: string = await this.wallet.getNonce(this.toNonceProvider);
+
+        const signedNonce: { v: number, r: string, s: string, transactionHash: string }
+            = await this.wallet.signNonce(nonce);
+
+        const webHookAttributes: WebHookAttributes = {
+            'signedNonce': signedNonce,
+            'nonce': nonce,
+            'to': this.contract.address,
+            'chain_id': this.contract.supportedChainsIDs[this.chain],
```

Comment:
Can we please use consistent casing?
`signedNonce` and `chain_id` are inconsistent casing in the same object